opendatahub-io/data-science-pipelines-operator
# WIP: Update Manifests for Argo v3.4.16

## The issue resolved by this Pull Request:
Resolves #TODO

## Description of your changes:
Updates argo manifests for use with argo v3.4.16

Restructures the `config/argo` directory to match argoproj/argo-workflows manifests for easier maintenance.  (The names and contents remain generally the same)


## Testing instructions
Run `make deploy`, wait for DSPO to become available, then deploy a simple DSPA.  Verify workflows still run

## Checklist
